### ISSUE TITLE
CIS-BP identifier mapping for Lambert et al

### ISSUE THREAD

**Issue by laserson** [vanheeringen-lab/gimmemotifs#200]
It seems like the Lambert et al Cell review is a very popular resource. Unfortunately, all the CIS-BP identifiers used in their supplementary tables reference a v1.94d, which seems publicly unavailable afaict. Matt Weirauch sent me a mapping between v1.94d and v2.00, which is publicly available and included with a gimmemotifs installation. Would it be off-topic to include this mapping file into the distribution?

**Comment by simonvh** [comment]
Yeah, I also use and reference that paper a lot. So I'm open to discuss this, however, at the moment I'm not too sure what you mean here. What would that look like, and what would the purpose or use-case be?

**Comment by laserson** [comment]
Currently, if I take any of the CIS-BP ids from the paper, there is no way to actually match them to any motifs. There is one additional tab-delim file you could distribute in gimmemotifs that would provide the mapping from the unusable identifiers to usable identifiers that work with the version of CIS-BP that's distributed in gimmemotifs.
